persistence/src/commonMain/kotlin/com/wire/kalium/persistence/dao/UserDAOImpl.kt

```diff
@@ -120,12 +120,19 @@ class UserDAOImpl(
         .map { entryList -> entryList.map(mapper::toModel) }
 
     override suspend fun getUserByQualifiedID(qualifiedID: QualifiedIDEntity): Flow<UserEntity?> {
-        return userQueries.selectByQualifiedId(qualifiedID)
+        return userQueries.selectByQualifiedId(listOf(qualifiedID))
             .asFlow()
             .mapToOneOrNull()
             .map { it?.let { mapper.toModel(it) } }
     }
 
+    override suspend fun getUsersByQualifiedIDList(qualifiedIDList: List<QualifiedIDEntity>): Flow<List<UserEntity>> {
```

Comment:
This function can be renamed to `observeUserById` since it return a flow

Reply:
I changed it to just return list, we can add one with flow when needed